index.html

```diff
@@ -0,0 +1,101 @@
+<html>
+  <head>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <script defer="defer" src="components/Tabs/Tabs.js"></script>
+
+    <header>
+      <div class="Navbar" id="navbar">
+        <div class="Navbar__title">Jesse Hood</div>
+        <div class="Navbar--links">
+          <div class="Tab Tab--selected" data-tab="1">
+            <div class="Tab__text">Home</div>
+          </div>
+          <div class="Tab" data-tab="2">
+            <div class="Tab__text">About me</div>
+          </div>
+          <div class="Tab" data-tab="3">
+            <div class="Tab__text">Projects</div>
+          </div>
+          <!--
+          <div class="Tab" data-tab="4">
+            <div class="Tab__text">Contact me</div>
+          </div>
+          -->
+        </div>
+      </div>
+    </header>
+    <main>
+      <div class="Card Card__parent Tab__window Tab__window--selected">

```

Comment:
No need for "Card__parent". That would be a child of "Card". You can just add the styling of "Card__parent" to "Card". Also, there are no modifiers at the element level. So, "Tab__window--selected" would just be "Tab__window-selected". I know, that can feel a little unintuitive.